TaskHouseUnitTests/WorkersControllerUnitTests.cs

```diff
@@ -0,0 +1,169 @@
+using System;
+using Xunit;
+using TaskHouseApi.Controllers;
+using TaskHouseApi.Repositories;
+using Microsoft.AspNetCore.Mvc;
+using TaskHouseApi.Model;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+
+namespace TaskHouseUnitTests
+{
+    public class WorkersControllerUnitTests
+    {
+        //var
+        WorkersController controller;
+        IWorkerRepository repo;
+
+        //Constructor
+      public WorkersControllerUnitTests()
+        {
+            repo = new FakeWorkerRepository();
+            controller = new WorkersController(repo);
+        }
+
+       
+        [Fact]
+        public async System.Threading.Tasks.Task Get_Worker_with_Id_UnitTest() 
+        { 
+           
+           Worker TestWorker =  new Worker() 
+                { 
+                    Id = 1, 
+                    Username = "1234", 
+                    Password = "+z490sXHo5u0qsSaxbBqEk9KsJtGqNhD8I8mVBdDJls=", //1234
+                    Email = "test@test.com", 
+                    FirstName = "Bob1", 
+                    LastName = "Bobsen1", 
+                    Salt = "upYKQSsrlub5JAID61/6pA=="
+                };
+
+
+           
+            var result = await controller.Get(TestWorker.Id);
+            var resultObjectResult = await controller.Get(TestWorker.Id) as ObjectResult;
+            var resultObject = resultObjectResult.Value as TaskHouseApi.Model.Worker;
+
+           
+            Assert.IsType<ObjectResult>(result);
+            Assert.Equal(TestWorker.Id, resultObject.Id);
+
+        }
+  
+        [Fact]
+        public async System.Threading.Tasks.Task Get_Worker_All_UnitTest() 
```

Comment:
Same, and seems to be going thoughout the file - you don't get anything to process from unitests, so you don't need to return anything